Sources/Swift/Tools/SentryLog.swift

```diff
@@ -26,7 +27,13 @@ class SentryLog: NSObject {
     @objc
     static func log(message: String, andLevel level: SentryLevel) {
         guard willLog(atLevel: level) else { return }
-        logOutput.log("[Sentry] [\(level)] \(message)")
+        
+        // We use the timeIntervalSinceReferenceDate because date format is
+        // expensive and we only care about the time difference between the
+        // log messages. We don't use system uptime because of privacy concerns
+        // see: NSPrivacyAccessedAPICategorySystemBootTime.
+        let time = self.dateProvider.date().timeIntervalSinceReferenceDate
```

Comment:
No change required, any reason you chose `timeIntervalSinceReferenceDate` over `timeIntervalSince1970`?

Reply:
No, no specific reason. Would you prefer `timeIntervalSince1970`, if yes why?

Reply:
No preferred reason, coming from devOPs perspective, most logging systems use epoch time as absolute time, so relative time to 1970, while `timeIntervalSinceReferenceDate` is relative to 2001.

As we do not submit the logs anywhere else, where the time could be interpreted wrongly, I don't see an issue with using `timeIntervalSinceReferenceDate`.

Reply:
Ahh, I'm working too long with the Apple ecosystem. timeIntervalSinceReferenceDate doesn't seem confusing to me anymore. Will switch to timeIntervalSince1970.